### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by alexshp1** [zh-plus/openlrc#4] [Q&A] [answered]
Hi, I wanted to use this repository, but I have this error, I would like you to tell me if I did something wrong. Thank you.
![image](https://github.com/zh-plus/Open-Lyrics/assets/121145845/0f3945d4-470d-4391-b4c0-1908f902496e)

**Comment by zh-plus** [comment, marked as answer]
I have resolved this issue in a recent commit. However, the updated version has not been released on PYPI yet. To install the latest version from GitHub, please use the following command:

```shell
pip install git+https://github.com/zh-plus/Open-Lyrics
```
